opm/parser/share/keywords/000_Eclipse100/W/WCONPROD

```diff
@@ -9,7 +9,7 @@
        {"name" : "RESV"         , "value_type" : "DOUBLE" , "default" : 0.0 , "dimension" : "LiquidSurfaceVolume/Time"},
        {"name" : "BHP"          , "value_type" : "DOUBLE" , "default" : 0.0 , "dimension" : "Pressure"},
        {"name" : "THP"          , "value_type" : "DOUBLE" , "default" : 0.0 , "dimension" : "Pressure"},
-       {"name" : "VFPTable"     , "value_type" : "INT"   , "default" : 0.0 , "comment":"The default is a state variable"},
+       {"name" : "VFP_TABLE"    , "value_type" : "INT"    , "default" : 0.0 , "comment":"The default is a state variable"},
```

Comment:
Two comments here:

1. The default value is obviously wrong - if it is `int` it should be `0`.
2. Now when we have a real implementation of the `VFPPROD` we can maybe implement the proper default behavior and remove the cryptic "The default is a state variable"?